examples/mm/clip/test_reward_model.py

```diff
@@ -0,0 +1,123 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pdb
+
+import numpy as np
+import torch
+import torch.multiprocessing as mp
+from omegaconf.omegaconf import OmegaConf
+from torch.nn import functional as F
+from torch.utils.data import DataLoader
+from tqdm import tqdm
+from transformers import AutoModel, AutoProcessor
+
+from nemo.collections.nlp.parts.megatron_trainer_builder import MegatronTrainerBuilder
+from nemo.core.config import hydra_runner
+from nemo.utils import logging
+from nemo.utils.exp_manager import exp_manager
+from nemo_aligner.data.mm.pickscore_dataset import build_train_valid_datasets
+from nemo_aligner.models.mm.stable_diffusion.image_text_rms import get_reward_model
+from nemo_aligner.utils.distributed import Timer
+
+
+@hydra_runner(config_path="conf", config_name="baseline")
+@torch.no_grad()
+def main(cfg) -> None:
+    logging.info("\n\n************** Experiment configuration ***********")
+    logging.info(f"\n{OmegaConf.to_yaml(cfg)}")
+
+    cfg.model.global_batch_size = cfg.trainer.devices * cfg.trainer.num_nodes * cfg.model.micro_batch_size
+
+    model = get_reward_model(cfg, cfg.model.micro_batch_size, cfg.model.global_batch_size).cuda()
+    model.eval()
+    batch_size = 250
```

Comment:
why hard coding batch size here?

Reply:
no particular reason, I just found 250 to be a good batch size for iterating quickly over the entire val set. I've changed it to `cfg.model.micro_batch_size`